index.html

```diff
@@ -595,6 +595,32 @@ <h3 id="profile-description-methodology">Methodology</h3>
     </section>
   </section>
 
+  <section id="device-categoriesdevices">
+    <h2>Device Categories</h2>	  
+<p>
+  In a deployment of WoT conforming to the <a href="https://www.w3.org/TR/wot-architecture/#architecture-abstract">WoT Abstract Architecture</a>
+  we see different device types with typical characteristics.
+  They range (sorted in the order of footprint and capabilities) 
+  from small embedded <em>node</em> devices to <em>gateways</em> or <em>hubs</em> 
```

Comment:
**small** does not imply resource constrained. **Constrained** or **resource constrained** embedded devices (like field devices) is more common.